src/client/components/MainHeader/SecondaryLinks.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon } from '@amsterdam/design-system-react';
```

Comment:
Beter in een apart component. Ik dacht dat het niet jouw stijl was maar laten we het bij deze vaker doen.

Reply:
Ik maak wel >1 componentjes in 1 file wanneer deze componenten niet geïmporteerd worden door andere componenten. Bij SecondaryLinks was dat het geval maar nu niet meer! Zoiets? :-)